.travis.yml

```diff
@@ -2,7 +2,7 @@ language: scala
 scala:
   - 2.11.6
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION validate publishLocal
+  - sbt ++${TRAVIS_SCALA_VERSION} coverage validate coverageReport coverageAggregate && bash <(curl -s https://codecov.io/bash) && sbt clean validate publishLocal ;
```

Comment:
Has sbt-scoverage found a way to support `coverageAggregate` in the same SBT invocation? The [README](https://github.com/scoverage/sbt-scoverage#multi-project-reports) still says it needs to be in a separate one.